### ISSUE TITLE
REPL: Up/Down arrow for command history in Linux (WSL) does not work.

### ISSUE THREAD

**Issue by snizar00** [microsoft/Power-Fx#1949]
Not sure if this (_up/down arrow for command history_) is the issue in Linux (WSL, windows Linux subsystem) or in the REPL but this (_up/down arrow for command history_) works in Windows CMD.

Details:
-> pac power-fx repl
>> 
>> Set(a, 5)
a: 5
true

>>
Up/Down arrow for the command history for the last command does not work.

**Comment by snizar00** [comment]
Left/Right arrows don't work either in Linux (WSL) to modify the formula/function. This is not the behavior in Windows CMD

**Comment by jorisdg** [comment]
The arrow key behavior in Windows is actually a feature of the windows CMD itself, not of the REPL or PAC CLI. I remember we talked about this feature explicitly but at least for the first release there's no plans to explicitly support/implement a behavior.

TL;DR; the Windows arrow keys behavior is a "free feature" of the CMD and not an explicit REPL feature.

**Comment by snizar007** [comment]
Thanks for clarification. This can be now closed.
